docs/advanced-cp4waiops-installation-option.md

```diff
@@ -0,0 +1,172 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Deploy CP4WAIOps Demo Environment In One Click Using GitOps](#deploy-cp4waiops-demo-environment-in-one-click-using-gitops)
+  - [Prepare Environment](#prepare-environment)
+  - [Install CP4WAIOps](#install-cp4waiops)
+    - [Installation Parameters](#installation-parameters)
+  - [Access Environment](#access-environment)
+    - [CP4WAIOps](#cp4waiops)
+    - [Robot Shop](#robot-shop)
+    - [Humio](#humio)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Deploy CP4WAIOps Demo Environment In One Click Using GitOps
+
+
+This tutorial will work you through the extremely easy steps to deploy demo environment for Cloud Pak for Watson AIOps (CP4WAIOps) using GitOps. It allows you to:
+
+- Deploy CP4WAIOps using custom profile, e.g.: extremely small profile in a sandbox with restricted resource.
+- Auto-configure integration with Humio, Kafka, Kubernetes, etc. as post-install step.
+- Deploy Robot Shop as sample application and other dependencies on the same cluster where CP4WAIOps runs.
+
+Althouth in this tutorial, I will use the extremely small profile to demonstrate the installation of CP4WAIOps, sample application, and dependencies, which is targeted for demo or PoC deployment rather than production deployment, the same idea can be applied to CP4WAIOps installation in production environment as well.
+
+This approach has been tested and verfied on CP4WAIOps 3.2.
+
+![](images/00-demo-env.png)
```

Comment:
May need to add the missing images if they are not ported from the cp4waiops-sandbox repo.

Reply:
Removed this readme and will commit a new PR to include the advanced installation option doc.